lib/util.js

```diff
@@ -41,7 +42,8 @@ exports.format = function(f) {
         return x;
     }
   });
-  for (var x = args[i]; i < len; x = args[++i]) {
+
```

Comment:
The line where I left this comment and the following line (lines 45 and 46 in the diff).